nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/DataSetExportStatusComputation.java

```diff
@@ -86,6 +86,7 @@ public void processRecord(ComputationContext context, String inputStreamName, Re
         BulkService service = Framework.getService(BulkService.class);
         if (isEndOfBatch(exportStatus)) {
             BulkCommand command = service.getCommand(exportStatus.getCommandId());
+            log.debug("Ending batch for {} " + exportStatus.getCommandId());
```

Comment:
Should be:
```suggestion
            log.debug("Ending batch for {}", exportStatus.getCommandId());
```
Same thing for line 113.